test/pom.xml

```diff
@@ -2,7 +2,7 @@
 <!--
 The MIT License
 
-Copyright (c) 2004-2009, Sun Microsystems, Inc., Kohsuke Kawaguchi
+Copyright (c) 2004-2018, Sun Microsystems, Inc., Kohsuke Kawaguchi, CloudBees, Inc.
```

Comment:
```suggestion
Copyright (c) 2004-2018, Sun Microsystems, Inc., Kohsuke Kawaguchi, CloudBees, Inc. and other Jenkins contributors
```

Reply:
No, as written in the PR description, this is just about doing the minimal file move, so the diff tab is pretty much useless. If you care to make any other changes, they can be filed separately.